frontend/src/pages/RecurringRunsManager.tsx

```diff
@@ -114,20 +114,22 @@ class RecurringRunsManager extends React.Component<RecurringRunListProps, Recurr
     );
   };
 
-  public _enabledCustomRenderer: React.FC<CustomRendererProps<boolean>> = (
-    props: CustomRendererProps<boolean>,
+  public _enabledCustomRenderer: React.FC<CustomRendererProps<V2beta1RecurringRunStatus>> = (
+    props: CustomRendererProps<V2beta1RecurringRunStatus>,
   ) => {
     const isBusy = this.state.busyIds.has(props.id);
     return (
       <BusyButton
-        outlined={props.value}
-        title={props.value === true ? 'Enabled' : 'Disabled'}
+        outlined={props.value === V2beta1RecurringRunStatus.ENABLED}
+        title={props.value === V2beta1RecurringRunStatus.ENABLED ? 'Enabled' : 'Disabled'}
```

Comment:
We call v2 listRecurringRuns() API only, which means that **no matter it is v1 or v2 recurring run, it will be in v2 format**. In addition, the `enabled` in v1 is change into `RecurringRunStatus` by the backend implementation. Therefore, the current logic is also applicable to v1 use case.

Reply:
To my knowledge: If this is V1 Recurring run, and we use V2 API to get the recurring run, how is `run parameters` being returned by the backend? Are `Run Parameters` being returned within `RuntimeConfig`?

Reply:
This one is to be determined. Need more information from the backend team, but I think the required field `id`, `name`, `enabled`, are preserved in corresponding field in v2 API response.